x-pack/plugins/infra/public/pages/metrics/hosts/components/table/filter_action.tsx

```diff
@@ -32,32 +36,52 @@ export const FilterAction = ({ selectedItemsCount, filterSelectedHosts }: Filter
   };
 
   return (
-    <EuiPopover
-      isOpen={isPopoverOpen}
-      closePopover={closePopover}
-      data-test-subj="bulkAction"
-      panelPaddingSize="s"
-      button={
+    <Container>
```

Comment:
To keep the consistency of the `emotion css` usage you can also use it here: 
```
    <div
      css={css`
        position: relative;
        height: ${euiTheme.size.l};
      `}
    >
```
Also, you are saving bundle size by not importing `euiStyled`, wdyt?

Reply:
yeah. good point.